updatecli/weekly.d/jenkinscontroller-agents-jnlp-container-jdk17.yaml

```diff
@@ -0,0 +1,52 @@
+---
+name: Bump JNLP Container Agent Image jenkins/inbound-agent (JDK17)
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  packerImageVersion:
+    kind: githubrelease
+    spec:
+      owner: "jenkins-infra"
+      repository: "packer-images"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+

```

Comment:
```suggestion

```

- If you open the YAML manifest in vscode (or any IDE supporting YAML schema validation), you should see a syntax error around "duplicated `sources` key".
- Executing the `updatecli diff` command sends the following error:

```
Loading Pipeline "updatecli/weekly.d/jenkinscontroller-agents-jnlp-container-jdk17.yaml"
ERROR: failed loading pipeline(s)
        * "updatecli/weekly.d/jenkinscontroller-agents-jnlp-container-jdk17.yaml" - yaml: unmarshal errors:
  line 25: mapping key "sources" already defined at line 16
```